packages/vite-plugin-cloudflare/src/cloudflare-environment.ts

```diff
@@ -148,6 +148,7 @@ export function createCloudflareEnvironmentOptions(
 			// We need to enable `emitAssets` in order to support additional modules defined by `rules`
 			emitAssets: true,
 			outDir: getOutputDirectory(userConfig, environmentName),
+			copyPublicDir: false,
```

Comment:
This is not wanted when we are assets-only, right?
But then `createCloudflareEnvironmentOptions()` is not called in that case, right?

Reply:
Yes, there is no Worker environment when it's assets only, just the `client` environment.